### ISSUE TITLE
Failed tests (10660): TestConvergence: newRaft: state.commit is out of range

### ISSUE THREAD

**Issue by tschottdorf** [cockroachdb/cockroach#3635]
The following test appears to have failed:

[#10660](https://circleci.com/gh/cockroachdb/cockroach/10660):

```
I0106 09:11:26.863210 1 storage/engine/rocksdb.go:106  opening rocksdb instance at "/data0.0"
I0106 09:11:26.865855 1 gossip/gossip.go:738  starting client to roach1.local:26257
I0106 09:11:26.886081 1 gossip/client.go:156  node 1 received 5 (5 fresh) info(s) from node 2
I0106 09:11:26.886226 1 gossip/server.go:211  @1: distantHops: 2 from 3
E0106 09:11:26.886991 1 raft/raft.go:843  [group 2] 1 state.commit 40 is out of range [37, 39]
panic: [group 2] 1 state.commit 40 is out of range [37, 39]

goroutine 1 [running]:
github.com/cockroachdb/cockroach/storage.(*raftLogger).Panicf(0xc820285f40, 0x15c2d40, 0x2b, 0xc8203e73c0, 0x4, 0x4)
	/go/src/github.com/cockroachdb/cockroach/storage/raft.go:124 +0x1a2
github.com/coreos/etcd/raft.(*raft).loadState(0xc8201049a0, 0x7, 0x0, 0x28, 0x0, 0x0, 0x0)
	/go/src/github.com/coreos/etcd/raft/raft.go:843 +0x2b3
github.com/coreos/etcd/raft.newRaft(0xc82026a828, 0x7fda073d6378)
	/go/src/github.com/coreos/etcd/raft/raft.go:221 +0x905
github.com/coreos/etcd/raft.NewRawNode(0xc82026a828, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
	/go/src/github.com/coreos/etcd/raft/rawnode.go:76 +0xba
github.com/cockroachdb/cockroach/storage.(*Replica).setReplicaID(0xc82024c820, 0xc800000001, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/storage/replica.go:291 +0x2d6
--
github.com/cockroachdb/cockroach/cli.Run(0xc82000a0b0, 0x9, 0x9, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/cli/cli.go:83 +0x7c
main.main()
	/go/src/github.com/cockroachdb/cockroach/main.go:37 +0xe6

goroutine 17 [syscall, locked to thread]:
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:1721 +0x1

goroutine 7 [chan receive]:
github.com/cockroachdb/cockroach/util/log.(*loggingT).flushDaemon(0x204fa00)
	/go/src/github.com/cockroachdb/cockroach/util/log/clog.go:1029 +0x64
created by github.com/cockroachdb/cockroach/util/log.init.1
	/go/src/github.com/cockroachdb/cockroach/util/log/clog.go:610 +0xbc

goroutine 21 [syscall]:
os/signal.loop()
	/usr/local/go/src/os/signal/signal_unix.go:22 +0x18
created by os/signal.init.1
	/usr/local/go/src/os/signal/signal_unix.go:28 +0x37

goroutine 22 [select]:
github.com/cockroachdb/cockroach/rpc.(*RemoteClockMonitor).MonitorRemoteOffsets(0xc8202831e0, 0xc8201edce0)
	/go/src/github.com/cockroachdb/cockroach/rpc/clock_offset.go:156 +0x742
github.com/cockroachdb/cockroach/server.NewServer.func1()
	/go/src/github.com/cockroachdb/cockroach/server/server.go:120 +0x31
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8201edce0, 0xc820283200)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x62

goroutine 23 [select]:
github.com/cockroachdb/cockroach/storage.(*StorePool).start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/store_pool.go:213 +0x1b6
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8201edce0, 0xc8202832e0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x62

goroutine 24 [select]:
github.com/cockroachdb/cockroach/util.NewFeed.func1()
	/go/src/github.com/cockroachdb/cockroach/util/feed.go:61 +0x157
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8201edce0, 0xc820284b10)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x62

goroutine 25 [select]:
github.com/cockroachdb/cockroach/kv.(*TxnCoordSender).startStats(0xc8202723c0)
	/go/src/github.com/cockroachdb/cockroach/kv/txn_coord_sender.go:214 +0xca5
github.com/cockroachdb/cockroach/kv.(*TxnCoordSender).(github.com/cockroachdb/cockroach/kv.startStats)-fm()
	/go/src/github.com/cockroachdb/cockroach/kv/txn_coord_sender.go:201 +0x20
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8201edce0, 0xc820284b60)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x62

goroutine 26 [select]:
github.com/cockroachdb/cockroach/sql.(*LeaseManager).RefreshLeases.func1()
	/go/src/github.com/cockroachdb/cockroach/sql/lease.go:617 +0xe0f
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8201edce0, 0xc8202882d0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x62

goroutine 27 [select]:
github.com/cockroachdb/cockroach/sql.newExecutor.func1()
	/go/src/github.com/cockroachdb/cockroach/sql/executor.go:97 +0x134
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8201edce0, 0xc820288420)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x62

goroutine 28 [IO wait]:
net.runtime_pollWait(0x7fda073d5878, 0x72, 0xc820016150)
	/usr/local/go/src/runtime/netpoll.go:157 +0x60
net.(*pollDesc).Wait(0xc820274d10, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc820274d10, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).accept(0xc820274cb0, 0x0, 0x7fda09c2b238, 0xc820011d40)
	/usr/local/go/src/net/fd_unix.go:408 +0x27c
net.(*TCPListener).AcceptTCP(0xc8200f4510, 0x8, 0x0, 0x0)
	/usr/local/go/src/net/tcpsock_posix.go:254 +0x4d
--
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8201edce0, 0xc820288720)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x62

goroutine 29 [chan receive]:
github.com/cockroachdb/cockroach/util.ListenAndServe.func2()
	/go/src/github.com/cockroachdb/cockroach/util/net.go:88 +0x5b
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8201edce0, 0xc820283620)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x62

goroutine 30 [chan receive]:
github.com/cockroachdb/cockroach/gossip.(*server).start.func2()
	/go/src/github.com/cockroachdb/cockroach/gossip/server.go:270 +0x5e
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8201edce0, 0xc8202836a0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x62

goroutine 31 [select]:
github.com/cockroachdb/cockroach/gossip.(*Gossip).bootstrap.func1()
	/go/src/github.com/cockroachdb/cockroach/gossip/gossip.go:549 +0x2b5
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8201edce0, 0xc8202836c0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x62

goroutine 32 [select]:
github.com/cockroachdb/cockroach/gossip.(*Gossip).manage.func1()
	/go/src/github.com/cockroachdb/cockroach/gossip/gossip.go:583 +0x32d
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8201edce0, 0xc8202836e0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x62

goroutine 33 [select]:
github.com/cockroachdb/cockroach/gossip.(*Gossip).maybeWarnAboutInit.func1()
	/go/src/github.com/cockroachdb/cockroach/gossip/gossip.go:689 +0x555
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8201edce0, 0xc820283700)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x62

goroutine 9 [select]:
github.com/cockroachdb/cockroach/gossip.(*client).gossip(0xc8200138b0, 0xc82024c270, 0xc8201edce0, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/gossip/client.go:232 +0x8fd
github.com/cockroachdb/cockroach/gossip.(*client).start.func1()
	/go/src/github.com/cockroachdb/cockroach/gossip/client.go:63 +0x162
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8201edce0, 0xc820130f00)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x62

goroutine 10 [select]:
github.com/cockroachdb/cockroach/rpc.(*Client).runHeartbeat(0xc820124580, 0x3b9aca00, 0x6fc23ac00, 0x4000000000000000, 0x0, 0x0, 0xc8200fede0)
	/go/src/github.com/cockroachdb/cockroach/rpc/client.go:318 +0x807
github.com/cockroachdb/cockroach/rpc.NewClient.func1()
	/go/src/github.com/cockroachdb/cockroach/rpc/client.go:148 +0x36
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8201edce0, 0xc820137080)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x62

goroutine 34 [chan receive]:
github.com/cockroachdb/cockroach/storage/engine.(*RocksDB).Open.func1(0xc82016bbd0)
	/go/src/github.com/cockroachdb/cockroach/storage/engine/rocksdb.go:123 +0x3a
created by github.com/cockroachdb/cockroach/storage/engine.(*RocksDB).Open
	/go/src/github.com/cockroachdb/cockroach/storage/engine/rocksdb.go:124 +0x490

goroutine 13 [IO wait]:
net.runtime_pollWait(0x7fda073d57b8, 0x72, 0xc820016150)
	/usr/local/go/src/runtime/netpoll.go:157 +0x60
net.(*pollDesc).Wait(0xc82012e300, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc82012e300, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).Read(0xc82012e2a0, 0xc820097000, 0x800, 0x800, 0x0, 0x7fda09c25050, 0xc820016150)
	/usr/local/go/src/net/fd_unix.go:232 +0x23a
net.(*conn).Read(0xc820026220, 0xc820097000, 0x800, 0x800, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:172 +0xe4
W0106 09:15:04.583881 514 rpc/server.go:309  rpc: write response failed: write tcp 127.0.0.1:43683->127.0.0.1:38635: write: connection reset by peer
W0106 09:15:04.584007 514 rpc/server.go:309  rpc: write response failed: write tcp 127.0.0.1:45370->127.0.0.1:39496: use of closed network connection
W0106 09:15:04.584313 514 rpc/server.go:309  rpc: write response failed: write tcp 127.0.0.1:45370->127.0.0.1:39496: use of closed network connection
W0106 09:15:04.584606 514 rpc/server.go:309  rpc: write response failed: write tcp 127.0.0.1:33397->127.0.0.1:56779: use of closed network connection
W0106 09:15:04.584894 514 rpc/server.go:309  rpc: write response failed: write tcp 127.0.0.1:33397->127.0.0.1:56779: use of closed network connection
--- FAIL: TestConvergence (0.38s)
	convergence_test.go:33: expected a fully-connected network within 20 cycles; took 21
FAIL
FAIL	github.com/cockroachdb/cockroach/gossip	0.587s
=== RUN   TestParseResolverSpec
--- PASS: TestParseResolverSpec (0.00s)
=== RUN   TestGetAddress
--- PASS: TestGetAddress (0.00s)
PASS
ok  	github.com/cockroachdb/cockroach/gossip/resolver	1.041s
?   	github.com/cockroachdb/cockroach/gossip/simulation	[no test files]
=== RUN   TestKeySorting
--- PASS: TestKeySorting (0.01s)
=== RUN   TestMakeKey
--
W0106 09:20:25.676029 748 storage/store.go:643  error gossiping system config: storage/replica.go:1527: node is stopping
I0106 09:20:50.874039 748 kv/range_cache.go:128  lookup range descriptor: key=/Table/501/0
I0106 09:20:50.874170 748 kv/range_cache.go:128  lookup range descriptor: key=/Meta2/"\xf9\x01\xf5\x88"
I0106 09:20:50.874358 748 kv/replica_slice.go:54  node 1 is not gossiped: gossip/gossip.go:222: unable to lookup descriptor for node 1: gossip/gossip.go:319: key "node:1" does not exist or has expired
W0106 09:20:50.874428 748 kv/dist_sender.go:572  no replica node addresses available via gossip
panic: test timed out after 5m0s

goroutine 1939 [running]:
testing.startAlarm.func1()
	/tmp/workdir/go/src/testing/testing.go:703 +0x16b
created by time.goFunc
	/tmp/workdir/go/src/time/sleep.go:129 +0x6e

goroutine 1 [chan receive, 4 minutes]:
testing.RunTests(0x1a472d0, 0x21084c0, 0x1f, 0x1f, 0x16cf101)
	/tmp/workdir/go/src/testing/testing.go:562 +0xafa
testing.(*M).Run(0xc820035f10, 0x171e3247c519ce5c)
	/tmp/workdir/go/src/testing/testing.go:494 +0xe5
github.com/cockroachdb/cockroach/util/leaktest.TestMainWithLeakCheck(0xc820035f10)
	/go/src/github.com/cockroachdb/cockroach/util/leaktest/leaktest.go:34 +0x2f
github.com/cockroachdb/cockroach/server_test.TestMain(0xc820035f10)
	/go/src/github.com/cockroachdb/cockroach/server/main_test.go:34 +0x2f
main.main()
	github.com/cockroachdb/cockroach/server/_test/_testmain.go:114 +0x20a

goroutine 17 [syscall, 4 minutes, locked to thread]:
runtime.goexit()
	/tmp/workdir/go/src/runtime/asm_amd64.s:1721 +0x1

goroutine 1912 [select]:
github.com/cockroachdb/cockroach/util/retry.(*Retry).Next(0xc8209530b0, 0x1)
	/go/src/github.com/cockroachdb/cockroach/util/retry/retry.go:106 +0x26c
github.com/cockroachdb/cockroach/kv.(*DistSender).sendChunk(0xc82023d6c0, 0x7f59a7703fe0, 0xc8200130c0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/kv/dist_sender.go:558 +0x44c
github.com/cockroachdb/cockroach/kv.(*DistSender).Send(0xc82023d6c0, 0x7f59a7703fe0, 0xc8200130c0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/kv/dist_sender.go:502 +0x30d
github.com/cockroachdb/cockroach/client.(*DB).send(0xc82049a140, 0xc8208fe9e0, 0x1, 0x1, 0x1, 0x61a614)
	/go/src/github.com/cockroachdb/cockroach/client/db.go:475 +0x22a
github.com/cockroachdb/cockroach/client.(*DB).(github.com/cockroachdb/cockroach/client.send)-fm(0xc8208fe9e0, 0x1, 0x1, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/client/db.go:446 +0x4c
--
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202b3260, 0xc8208dc600)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 19 [chan receive]:
github.com/cockroachdb/cockroach/util/log.(*loggingT).flushDaemon(0x2455120)
	/go/src/github.com/cockroachdb/cockroach/util/log/clog.go:1029 +0x76
created by github.com/cockroachdb/cockroach/util/log.init.1
	/go/src/github.com/cockroachdb/cockroach/util/log/clog.go:610 +0x117

goroutine 1926 [chan receive, 4 minutes]:
github.com/cockroachdb/cockroach/util.ListenAndServe.func2()
	/go/src/github.com/cockroachdb/cockroach/util/net.go:88 +0x7b
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202b3260, 0xc8209645c0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 1902 [chan receive, 4 minutes]:
github.com/cockroachdb/cockroach/storage/engine.(*RocksDB).Open.func1(0xc8208fd310)
	/go/src/github.com/cockroachdb/cockroach/storage/engine/rocksdb.go:123 +0x5b
created by github.com/cockroachdb/cockroach/storage/engine.(*RocksDB).Open
	/go/src/github.com/cockroachdb/cockroach/storage/engine/rocksdb.go:124 +0x581

goroutine 1923 [select, 4 minutes]:
github.com/cockroachdb/cockroach/util.NewFeed.func1()
	/go/src/github.com/cockroachdb/cockroach/util/feed.go:61 +0x1b6
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202b3260, 0xc820250560)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 1929 [select, 2 minutes]:
github.com/cockroachdb/cockroach/storage.(*Store).startGossip.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:621 +0x447
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202b3260, 0xc820964620)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 1930 [select, 2 minutes]:
github.com/cockroachdb/cockroach/storage.(*Store).startGossip.func2()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:640 +0x447
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202b3260, 0xc820964640)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 1916 [select, 4 minutes]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:319 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202b3260, 0xc8208dc680)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 1911 [select, 4 minutes]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:319 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202b3260, 0xc8208dc5e0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 1913 [select, 4 minutes]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:319 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202b3260, 0xc8208dc620)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 1927 [select]:
github.com/cockroachdb/cockroach/storage.(*Store).processRaft.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:1614 +0xb0c
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202b3260, 0xc8209bce80)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 1889 [IO wait, 4 minutes]:
net.runtime_pollWait(0x7f59a7703710, 0x72, 0xc820038b70)
	/tmp/workdir/go/src/runtime/netpoll.go:157 +0x63
net.(*pollDesc).Wait(0xc82023d480, 0x72, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:73 +0x56
net.(*pollDesc).WaitRead(0xc82023d480, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:78 +0x44
net.(*netFD).accept(0xc82023d420, 0x0, 0x7f59a5e95408, 0xc820954b80)
	/tmp/workdir/go/src/net/fd_unix.go:408 +0x2f6
net.(*TCPListener).AcceptTCP(0xc8200264a0, 0x8, 0x0, 0x0)
	/tmp/workdir/go/src/net/tcpsock_posix.go:254 +0x77
--
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202b3260, 0xc8202951d0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 1925 [IO wait, 4 minutes]:
net.runtime_pollWait(0x7f59a770d438, 0x72, 0x7f59a76f9050)
	/tmp/workdir/go/src/runtime/netpoll.go:157 +0x63
net.(*pollDesc).Wait(0xc82011aa00, 0x72, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:73 +0x56
net.(*pollDesc).WaitRead(0xc82011aa00, 0x0, 0x0)
	/tmp/workdir/go/src/net/fd_poll_runtime.go:78 +0x44
net.(*netFD).accept(0xc82011a9a0, 0x0, 0x7f59a5e95408, 0xc8208dc080)
	/tmp/workdir/go/src/net/fd_unix.go:408 +0x2f6
net.(*TCPListener).AcceptTCP(0xc8200fe078, 0x60cec0, 0x0, 0x0)
	/tmp/workdir/go/src/net/tcpsock_posix.go:254 +0x77
--
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202b3260, 0xc820903020)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 1914 [select, 4 minutes]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:319 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202b3260, 0xc8208dc640)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 1915 [select, 4 minutes]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:319 +0x427
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202b3260, 0xc8208dc660)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 1922 [chan receive, 4 minutes]:
github.com/cockroachdb/cockroach/util.ListenAndServe.func2()
	/go/src/github.com/cockroachdb/cockroach/util/net.go:88 +0x7b
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202b3260, 0xc820954920)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 1928 [select, 4 minutes]:
github.com/cockroachdb/cockroach/storage.(*Store).Start.func3()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:556 +0x15d
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202b3260, 0xc820964600)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 1917 [select, 4 minutes]:
github.com/cockroachdb/cockroach/storage.(*replicaScanner).waitAndProcess(0xc820a20c60, 0xece1ed269, 0x284f34ff, 0x2454ba0, 0xc820198300, 0xc8202b3260, 0xc8201c8000, 0xc8203ac1c0)
	/go/src/github.com/cockroachdb/cockroach/storage/scanner.go:176 +0x641
github.com/cockroachdb/cockroach/storage.(*replicaScanner).scanLoop.func1.1(0xc8201c8000, 0xc820295b30)
	/go/src/github.com/cockroachdb/cockroach/storage/scanner.go:217 +0xb0
github.com/cockroachdb/cockroach/storage.(*storeRangeSet).Visit(0xc820295bc0, 0xc8208f63c0)
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:225 +0x45c
github.com/cockroachdb/cockroach/storage.(*replicaScanner).scanLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/scanner.go:219 +0x452
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202b3260, 0xc8208dc6a0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:89 +0x60
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:90 +0x70

goroutine 1901 [semacquire, 4 minutes]:
sync.runtime_Syncsemacquire(0xc82025cf90)
	/tmp/workdir/go/src/runtime/sema.go:237 +0x201
sync.(*Cond).Wait(0xc82025cf80)
	/tmp/workdir/go/src/sync/cond.go:62 +0xb3
github.com/cockroachdb/cockroach/util/stop.(*Stopper).Quiesce(0xc8202b3260)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:266 +0x2a3
github.com/cockroachdb/cockroach/util/stop.(*Stopper).Stop(0xc8202b3260)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:212 +0x1df
github.com/cockroachdb/cockroach/server.TestCorruptedClusterID(0xc8203cb440)
	/go/src/github.com/cockroachdb/cockroach/server/node_test.go:278 +0xec2
testing.tRunner(0xc8203cb440, 0x21085b0)
	/tmp/workdir/go/src/testing/testing.go:456 +0xdd
created by testing.RunTests
	/tmp/workdir/go/src/testing/testing.go:561 +0xaa4
FAIL	github.com/cockroachdb/cockroach/server	300.076s
=== RUN   TestNodeStatusMonitor
--- PASS: TestNodeStatusMonitor (0.01s)
=== RUN   TestNodeStatusRecorder
--- PASS: TestNodeStatusRecorder (0.02s)
=== RUN   TestRuntimeStatRecorder
I0106 09:16:25.074586 760 server/status/runtime.go:137  runtime stats: 11 goroutines, 3.04MiB active, 50000000.00cgo/sec, 603720.00/384190.00 %(u/s)time, 9127.30 %gc (2x)
--- PASS: TestRuntimeStatRecorder (0.00s)
=== RUN   TestNodeEventFeed
--- PASS: TestNodeEventFeed (0.00s)
=== RUN   TestServerNodeEventFeed

```
Please assign, take a look and update the issue accordingly.

**Comment by tschottdorf** [comment]
maybe related to #3611

**Comment by bdarnell** [comment]
This failure looks miscategorized. The `1 state.commit 40 is out of range [37, 39]` failure occurs on one of the nodes in `TestChaos`, so this part is a duplicate of #3640. The `TestConvergence` failure is one that we've already addressed by increasing the cycle count. There's also a `TestCorruptedClusterID` failure which was fixed in #3649.
